spark/src/main/kotlin/com/adevinta/spark/components/toggles/CheckBox.kt

```diff
@@ -35,6 +35,7 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import com.adevinta.spark.InternalSparkApi
 import com.adevinta.spark.PreviewTheme
+import com.adevinta.spark.components.IntentColor
```

Comment:
Is this import unused ?

Reply:
I ran spotlessApply, I think it was removed :-)